app/assets/stylesheets/components/_cards.scss

```diff
@@ -0,0 +1,47 @@
+// this has to be moved to container later, put it here because of conflicts
+.arts-container {
+  display: flex;
+  justify-content: space-around;
+  margin-top: 100px;
+  margin-bottom: 20px;
+    a {
+      text-decoration: none;
+      color: white !important;
+    }
+}
+
+.art-card {
+  display: flex;
+  flex-direction: column;
+}
+
+.art-card-img {
+  box-sizing: border-box;
+  border-radius: 5%;
+  object-fit: cover;
+  height: 300px;
+  width: 300px;
+  margin-bottom: 5px;
+}
+
+.art-card-body {
+  margin-left: 5px;
+}
+
+.art-card-title {
+  font-weight: bold;
+}
+
+.art-card-text {
+  color: lightslategrey;
+}
+
+.favorites-gold {
+  color: $gold;
+}
+
+.favorites-location-div {
+  float: right;
+  margin: 20px;
+  font-size: 20px;
+}
```

Comment:
This can be removed as will be part of the favourite PR later

Reply:
I will remove this later

Reply:
You can also change the color of the art-card-price to light gray and the underscore in the name for a dash